src/controller.rs

```diff
@@ -403,14 +403,57 @@ fn pod_template(config: &WasmCloudHostConfig, _ctx: Arc<Context>) -> PodTemplate
         ..Default::default()
     }];
 
-    let mut volume_mounts = vec![VolumeMount {
+    let mut nats_volume_mounts = vec![VolumeMount {
         name: "nats-config".to_string(),
         mount_path: "/nats/nats.conf".to_string(),
         read_only: Some(true),
         sub_path: Some("nats.conf".to_string()),
         ..Default::default()
     }];
 
+    let mut wasm_volume_mounts: Vec<VolumeMount> = vec![];
+    if let Some(certificates) = &config.spec.certificates {
+        if let Some(authorities) = &certificates.authorities {
+            for (i, authority) in authorities.iter().enumerate() {
+                // configmaps
+                if let Some(configmap) = &authority.config_map_ref {
+                    let volume_name = format!("cert-authority-{}", i);
+                    let volume_path =
+                        format!("/wasmcloud/certificates/authorities/{}", volume_name);
+                    match discover_configmap_certificates(
+                        config.namespace().unwrap(),
```

Comment:
In theory `unwrap()` is probably fine here, but we should consider something that has a safe fallback, such as `unwrap_or_default()` even though this will most likely always be set (I can't imagine a scenario where it wouldn't be, but alas).

I know we have a lot of `unwrap()`s around, we need go through and fix those up, because it's not the pattern we want to encourage.

Reply:
makes sense. addressing it